src/Microsoft.AspNetCore.SignalR/HubEndPoint.cs

```diff
@@ -36,19 +36,19 @@ public class HubEndPoint<THub, TClient> : EndPoint, IInvocationBinder where THub
         private readonly HubLifetimeManager<THub> _lifetimeManager;
         private readonly IHubContext<THub, TClient> _hubContext;
         private readonly ILogger<HubEndPoint<THub, TClient>> _logger;
-        private readonly InvocationAdapterRegistry _registry;
         private readonly IServiceScopeFactory _serviceScopeFactory;
 
+        // TODO: Configuration and Protobuf
+        private readonly JsonHubProtocol _protocol = new JsonHubProtocol();
```

Comment:
We likely will when we have DI-based hub argument serialization, but I didn't want to jump the gun on that when we don't intend for users to be able to replace the outer Hub Protocol logic.